### ISSUE TITLE
cabal complains about missing .cabal files

### ISSUE THREAD

**Issue by step21** [haskell/cabal#7048]
On upgrading cabal and installing packages, with v2 commands, some packages fail with supposedly not cabal file being present. On manual inspection the file is there.

**To Reproduce**

```
$ cabal v2-update
$ cabal v2-install cabal-install
```

**Expected behavior**
Packages are downloaded and installed properly.

**System information**
 - centos 6/7
 - cabal 2.4.10, ghc 8.2.2

**Additional context**
Logs https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=207172&view=logs&jobId=517fe804-fa30-5dc2-1413-330699242c05&j=517fe804-fa30-5dc2-1413-330699242c05&t=c10fa5f2-fdf6-5338-3bdb-c4bea7c23412

**Comment by gbaz** [comment]
logs are dead, can't reproduce? also looks to be an ancient cabal. going to close this for now.

**Comment by step21** [comment]
Logs are always deleted after some time. I restarted CI and will update with new logs. The old versions are necessary I think to bootstrap. As there are for some Archs no usable binary packages of newer versions (at least last time I checked) I would need to bootstrap with the ghc / cabal etc of the CI system, which in this case was centos 7.
Other attempts are here: https://github.com/conda-forge/ghc-feedstock/pull/16 https://github.com/conda-forge/ghc-feedstock/pull/14
